Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.13 AS builder
```

Comment:
There has been a discussion about this issue on #14. I would keep it out of this PR.